app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,105 @@
+<section class="hero">
+  <div>
+    <h3><%= @recipe.name%></h3>
+    <div>
+      <div class="recipe-details">
+        <div class="recipes">
+          <div class="recipe-public">
+            <p> Public: </p>
+              <%= form_with url:recipe_path(@recipe.id), :method => :patch do |f| %>
+                <%= f.check_box :public, checked: @recipe.public, onchange: "this.form.submit();" %>
+              <% end %>
+          </div>
+          <p> Preparation time: <%= @recipe.preparation_time %> min</p>
+        </div>
+        <p> Cooking time: <%= @recipe.cooking_time %> min </p>
+        <div class="recipe-buttons">
+            <a id="openRecipeModal">Generate shopping list</a>
+            <a id="openInventoryModal">Add ingredient</a>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <% if @recipe.recipe_foods.empty? %>
+    <p>No ingredients added yet.</p>
+  <% else %>
```

Comment:
- [ ] Clicking on "Add ingredient" should lead to the form that allows the user to add new food, if the user is the owner of the recipe. Currently, this is not the case, rather the form for selecting Inventory items pops up. Kindly fix this.


Video Cast
[cinnamon-2023-11-26T001731+0200.webm](https://github.com/firepicaso/ror-recipe-app/assets/81932478/444298a8-d417-492f-a9b2-b16065849c7b)


#### Note that you need to swap the functionalities of the 2 buttons. clicking on ` Add Ingredient` should show the `Add recipe food` popup and clicking the `Generate shopping list` button should show the inventory pop up.